dev/sg/sg_ci.go

```diff
@@ -412,10 +459,22 @@ From there, you can start exploring logs with the Grafana explore panel.
 				}
 				return writePrettyMarkdown(out)
 			},
-		}},
+		},
+		},
 	}
 )
 
+func getBuildTypes() string {
```

Comment:
```suggestion
func getAllowedBuildTypeArgs() string {
```

an additional improvement, I think we should let this be:

```suggestion
func getAllowedBuildTypeArgs() []string {
```

and further above, change the help text to:

```go
ShortUsage: fmt.Sprintf("sg ci build [%s]", strings.Join(getAllowedBuildTypeArgs(), "|"))
```

Then we can also use `getAllowedBuildTypeArgs` to validate the provided argument, and error out if the argument is not allowed. If we want to be extra smart we can provide suggestions 😛